core/arch/arm/mm/sp_mem.c

```diff
@@ -17,6 +16,138 @@ static bitstr_t bit_decl(share_bits, NUM_SHARES);
 /* mem_shares stores all active FF-A shares. */
 SLIST_HEAD(sp_mem_head, sp_mem);
 static struct sp_mem_head mem_shares = SLIST_HEAD_INITIALIZER(sp_mem_head);
+const struct mobj_ops mobj_sp_ops;
+
+struct mobj_sp {
+	struct mobj mobj;
+	paddr_t *pages;
```

Comment:
How about making this a `paddr_t pages[]` instead? We do use that patter at various places.

Reply:
Done